glossary.yml

```diff
@@ -1052,6 +1070,10 @@ entries:
     description: |-
       The U.S. Government Accountability Office is known as "the investigative arm of Congress" and  "the congressional watchdog." GAO supports the Congress in meeting its constitutional responsibilities and helps improve the performance and accountability of the Federal Government for the benefit of the American people.
 
+  GMT:
```

Comment:
There's no term "General Management Team" yet, so this validation will fail.

Reply:
@beechnut I don't have a definition to pull from on this one. Can 'description' be null?

Reply:
nvm. resolved.